Backend/src/main/java/idealab/api/service/AuditUtil.java

```diff
@@ -0,0 +1,65 @@
+package idealab.api.service;
+
+import idealab.api.dto.response.PrintJobAuditModel;
+import idealab.api.model.ColorType;
+import idealab.api.model.PrintJob;
+import org.hibernate.envers.AuditReader;
+import org.hibernate.envers.AuditReaderFactory;
+import org.hibernate.envers.DefaultRevisionEntity;
+import org.hibernate.envers.RevisionType;
+import org.hibernate.envers.query.AuditQuery;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@PersistenceContext
+public class AuditUtil {
```

Comment:
I dont like this being called a "util," it has a specific purpose. Can we call this the AuditService?

Reply:
I updated to AuditServices.